api/alembic/versions/d9c05cb16869_add_combustible_area_data_to_advisory_.py

```diff
@@ -0,0 +1,32 @@
+"""Add combustible area data to advisory_shapes
+
+Revision ID: d9c05cb16869
+Revises: 0669994d4089
+Create Date: 2022-09-09 15:52:22.162223
+
+"""
+from alembic import op
+import asyncio
+from app.auto_spatial_advisory.calculate_combustible_land_area import calculate_combustible_area_by_fire_zone
+
+
+# revision identifiers, used by Alembic.
+revision = 'd9c05cb16869'
+down_revision = '0669994d4089'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(loop)
+    zone_areas = loop.run_until_complete(calculate_combustible_area_by_fire_zone())
```

Comment:
This won't work - you're opening a new session in calculate_combustible_area_by_fire_zone which is going to be blocked by the transaction alembic has started.